### ISSUE TITLE
Expired (TTL-wise) accounts do expire but it's not shown on accountInfo

### ISSUE THREAD

**Issue by speed47** [ovh/the-bastion#329]
see https://github.com/ovh/the-bastion/issues/313#issuecomment-1184158231

**Comment by speed47** [comment]
```
slesimpl@dev(master)> adminSudo -- --sudo-as expi30sec --sudo-cmd info
╭──ac777d06bec9───────────────────────────────────────the-bastion-3.09.00-rc3───
│ ▶ launching a bastion command or connection, impersonating another user
├───────────────────────────────────────────────────────────────────────────────
│ ❗ ADMIN SUDO: slesimpl, you'll now impersonate expi30sec, this has been logged.
│ 
│ ⛔ Sorry expi30sec, your account has expired.
│ Command exited with status 121
╰────────────────────────────────────────────────────────────────</adminSudo>───
slesimpl@dev(master)> accountInfo --account expi30sec
╭──ac777d06bec9───────────────────────────────────────the-bastion-3.09.00-rc3───
│ ▶ account information
├───────────────────────────────────────────────────────────────────────────────
│ This account has access to the following restricted commands:
│ (none)
│ 
│ This account is active
│ This account is not expired
│ As a consequence, this account can connect to this bastion

```

The terminology here can be confused, we need to change that.
In the context of `accountInfo`:
- `active` is whether this account has been declared active or inactive by an external system (see https://ovh.github.io/the-bastion/administration/configuration/bastion_conf.html#accountexternalvalidationprogram)
- `expired` is whether this account has logged-in recently enough to not be auto-expired (https://ovh.github.io/the-bastion/administration/configuration/bastion_conf.html#accountmaxinactivedays)
- The TTL status is not shown here, it should be. This also means that the sentence `this account can connect to this bastion` is incorrect in this case, as the TTL has expired

**Comment by speed47** [comment]
Merged and released in v3.09.00
